SETUP/tests/jsUnit.js

```diff
@@ -14,15 +12,19 @@ const assert = require('assert').strict;
             new firefox.ServiceBuilder(
                 `./node_modules/geckodriver/geckodriver${process.platform === 'win32' ? '.exe' : ''}`))
         .build();
+    let succeeded = false;
     try {
-        await driver.get(`file:///${process.cwd()}/SETUP/tests/qunit.html`);
+        await driver.get(`file:///${process.cwd()}/SETUP/tests/manual_web/qunit.html`);
```

Comment:
qunit tests aren't manual tests -- they don't require a user to do anything, they're part of our CI/CD process. Only tests that require a user to manually do something to validate it should be in `manual_web`.

Reply:
That was my opinion as well. You can load the file from file:// and not apache if you want to test locally. We should move it under test in my opinion and avoid duplicate.

Reply:
Moved tests out of manual tests. Had to put in htaccess to allow running manually.